sagemaker-pipelines/time_series_forecasting/amazon_forecast_pipeline/sm_pipeline_with_amazon_forecast.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>Style nit: preferred to use the name of a doc rather than "here" as link text. In this case, "View and Compare Amazon SageMaker Experiments, Trials, and Trial Components", "Manage Machine Learning with Amazon SageMaker Experiments"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>